tools/convert_cr_to_oc.py

```diff
@@ -21,21 +22,21 @@ def convertname(root, files, name):
 		os.unlink(os.path.join(root, "Names.txt"))
 		r = "\"$Name$\""
 	else:
-		print root, name
-	for lang, localname in names.iteritems():
+		print(root, name)
+	for lang, localname in names.items():
 		if "StringTbl" + lang + ".txt" in files:
-			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"rb").read().splitlines()
+			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"r").read().splitlines()
```

Comment:
py3 still supports binary mode when opening files, and your changes will most likely break Windows compatibility...